contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,130 @@
+# See readme.md for instructions on running this code.
+from github import GithubConn
+import requests
+import json
+
+
+class GitHubHandler(object):
+    '''
+    This plugin facilitates allows you to comment on a GitHub issue, under a certain repository.  It
+    looks for messages starting with '@comment' or '@github-comment.
+    '''
```

Comment:
I'm also wondering if `@github-comment` isn't a tad too long to type and there is a higher risk of typos - maybe `@gcomment` would work better?